### PR TITLE
Epic/registration

### PR DESCRIPTION
**Pull request by arcvelit**
Merging Salma's code of registration page with main

### CODE REVIEW

File: app/src/App.tsx [hunk 1]
```diff
@@ -12,25 +12,16 @@ import Home from "./home/home";
...
+import {Register} from "./Register/Register";
+import './Register/Register.css'
+
 function App() {
-  const user = true; // if user is login, this value is true, for now the value is always true
+  const user = false; // if user is login, this value is true, for now the value is always true
   return (
     <>
       <Header />
-      <Switch>
```
Comment (Razvan-Alexandru) [inline app/src/App.tsx:20]: This should not be modified

File: app/src/LoginPage/LoginPage.js [hunk 2]
```diff
@@ -1,43 +0,0 @@
-import React, { useState } from 'react';
```
Comment (Razvan-Alexandru) [inline app/src/LoginPage/LoginPage.js:1]: Why is this page deleted and re-created with a different extension? Also what modification happened here?

File: app/src/components/Header/Header.jsx [hunk 3]
```diff
@@ -1,25 +1,18 @@
...
+import './Header.css';
 
 export default function Header() {
   return (
-    <div class="header">
-        <div class="logo">
+    <div className="header">
+        <div className="logo">
             <h2>Hired</h2>
-            <svg class="logo-image" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"/></svg>
```
Comment (Razvan-Alexandru) [inline app/src/components/Header/Header.jsx:8]: What are all these modifications?

**Review by Razvan-Alexandru — CHANGES_REQUESTED** [review]
Make sure your branch is up to date with main. All these changes should not happen

File: app/src/components/Header/Header.jsx (hunk 3 above)
Comment (wam82) [reply app/src/components/Header/Header.jsx:8]: This appears to be a modification in the image in the header, it shouldn't break the code. @SalmaAl22 Can you confirm?

**Comment by SalmaAl22** [comment]
guys you told me it's been taken care of, the linking to database issue, but why isn't merged to main yet?

**Comment by wam82** [comment]
Because there are a couple changes that were made. I'll close to pull request. Assume it to be completed

**Comment by SalmaAl22** [comment]
There's is still no registration file in the main!

**Pull request closed without merging**